packages/cli/e2e/__tests__/deploy.spec.ts

```diff
@@ -50,4 +50,18 @@ describe('deploy', () => {
     })
     expect(result.stderr).toBe('')
   })
+
+  it('Shouldn\'t include a testOnly check', () => {
+    const result = runChecklyCli({
+      args: ['deploy', '--preview'],
+      apiKey: config.get('apiKey'),
+      accountId: config.get('accountId'),
+      directory: path.join(__dirname, 'fixtures/test-only-project'),
+      env: { PROJECT_LOGICAL_ID: projectLogicalId },
```

Comment:
Nit: 

This PR currently doesn't use the `PROJECT_LOGICAL_ID` env var. It's there in the other test since it's read here: https://github.com/checkly/checkly-cli/blob/d102c1b043d878c1335c28b0b996680ae7bdd204/packages/cli/e2e/__tests__/fixtures/deploy-project/checkly.config.ts#L3

Currently the test doesn't do any cleanup of the project. The checks will just remain in the E2E account and keep running. Maybe it makes sense to add some cleanup similar to what we have with the other project with `cleanupProjects`:

https://github.com/checkly/checkly-cli/blob/d102c1b043d878c1335c28b0b996680ae7bdd204/packages/cli/e2e/__tests__/deploy.spec.ts#L41

On the other hand, adding some cleanup isn't super important. The test looks good.

Reply:
@clample , thanks, I didn't realize about the `env.PROJECT_LOGICAL_ID`. Anyway, I'm using `deploy --preview` in the test.